guardian/support-frontend
# Checkout - Transaction Costs Changes for V2

## What are you doing in this PR?

Visual changes for a second run of the cover transaction costs (ab-test)

VariantA
- Copy

VariantB
- Copy
- Move Up into choice cards
- Include Total Summary (initially hidden until checked)
- Hide on `Other` Amount with no associated amount 

Further to these PR's ->
https://github.com/guardian/support-frontend/pull/6277
https://github.com/guardian/support-frontend/pull/6282

[**Trello Card**](https://trello.com/c/3Nq00TOL/1084-build-new-design-for-new-test-variants-cover-transaction-costs)

## How to test

Go to ->
1. `uk/contribute/checkout?selected-contribution-type=one_off#ab-coverTransactionCost=variantA` 
2. `uk/contribute/checkout?selected-contribution-type=one_off#ab-coverTransactionCost=variantB` 
3. `uk/contribute/checkout?selected-contribution-type=one_off#ab-coverTransactionCost=control` 

- Set checkbox to add 4%. Check checkbox. 
- Amount in CTA changes to add 4% (eg choose 100 see 104). Selected choice card remains the same. 
- Pay. 
- Thank you page shows amount + 4%.

## Screenshots
|variantA|variantB|
|-----|-----|
|<img width="1174" alt="image" src="https://github.com/user-attachments/assets/d5b656b8-ec1c-471e-a244-069c9638b2ad">|<img width="1174" alt="image" src="https://github.com/user-attachments/assets/8c99fdc1-e22c-4c61-bf90-5e6bbafd2015">|
